main.go

```diff
@@ -255,11 +255,11 @@ func cloneusergists(ctx context.Context, client *github.Client, user string) err
 		fmt.Println(gisturl)
 
 		//cloning the individual user gists
-		func(gisturl string, userGist *github.Gist, user string, usergistclone *sync.WaitGroup) {
```

Comment:
why did we remove this part?

Reply:
Disregard this part; it was a mistake. The only relevant part is 533:565. I committed the modified parts back in.